bindings/matrix-sdk-ffi/src/client_builder.rs

```diff
@@ -102,7 +103,24 @@ impl ClientBuilder {
             let data_path = PathBuf::from(base_path).join(sanitize(username));
             fs::create_dir_all(&data_path)?;
 
-            inner_builder = inner_builder.sled_store(data_path, builder.passphrase.as_deref());
+            let mut state_store =
+                matrix_sdk_sled::SledStateStore::builder().path(data_path.to_owned());
+
+            if let Some(passphrase) = builder.passphrase.as_deref() {
+                state_store = state_store.passphrase(passphrase.to_owned());
+            }
+
+            let state_store = state_store.build()?;
+
+            let crypto_store = RUNTIME.block_on(matrix_sdk_sqlite::SqliteCryptoStore::open(
```

Comment:
But also, why does the sled one have a builder and this one doesn't? 🤔

Reply:
The state store is a bit more complex, i.e. you have the ability to expand the state store into a crypto store which uses the same underlying database.

In any case, once we have a SQLite state store as well, we'll add the ability to configure the SQLite store on the client builder and you won't have to access this low level interface.